plugins/ltac/g_class.mlg

```diff
@@ -86,12 +86,16 @@ END
 
 (** Compatibility: typeclasses eauto has 8.5 and 8.6 modes *)
```

Comment:
FWIW, this comment is outdated and could be removed (I don't know why it was located there in the first place).

Reply:
OK, I will remove.